package.json

```diff
@@ -4,12 +4,13 @@
   "description": "Utilities for testing babel plugins",
   "main": "dist/index.js",
   "engines": {
-    "node": ">=8",
+    "node": ">=10.14",
```

Comment:
```suggestion
    "node": ">=10.13",
```

Reply:
10.13.0 is supported by prettier 2: https://prettier.io/blog/2020/03/21/2.0.0.html